src/status_im/ui/screens/wallet/transactions/subs.cljs

```diff
@@ -57,3 +62,16 @@
       postponed (into postponed)
       pending   (into pending)
       completed (into completed))))
+
+(reg-sub :wallet.transactions/transaction-details
+  :<- [:wallet.transactions/transactions]
+  (fn [transactions]
+    (let [{:keys [gas-used gas-price timestamp] :as transaction} (first (:inbound transactions))]
+      (assoc transaction
+             :cost (money/wei->ether (money/fee-value gas-used gas-price))
+             :date (datetime/timestamp->long-date timestamp)))))
+
+(reg-sub :wallet.transactions.details/confirmations
+  :<- [:wallet.transactions/transaction-details]
+  (fn [transaction-details]
+    3 #_(:confirmations transaction-details)))
```

Comment:
WIP ?

Reply:
yes for now design is ready but some work is required on the data-binding side